src/gui/ObsListCreateEditDialog.cpp

```diff
@@ -119,7 +119,7 @@ void ObsListCreateEditDialog::createDialogContent()
     } else {
         // case of edit mode
         isCreationMode = false;
-        ui->stelWindowTitle->setText ( "Observing list modification mode" );
+        ui->stelWindowTitle->setText ( "Observing list editor" );
```

Comment:
@axd1967 I changed "modification mode into "editor mode". Is it ok for you ?

Reply:
_(If you allow me to be honest, it feels slightly wrong? Rest assured, it makes ME feel bad to have to write this, because it's so much easier to criticize (even in a positive way) than to develop a working solution; so please bear with me. The core issue, having different bookmark lists, remains a significant step forward because users had to juggle with a lot of bookmark files)._

Either there is _one_ window that has several modes (in which case the user is shown the mode), or there are different windows that each do a specific (and probably complex) task.

In this case, the user is interacting with the **observing list editor**. Another indication that there is something smelly is that the user switches to a different mode _and to a new dialog_.


I think that the real problem is due to the near-identical UI of two slightly different windows (which has a heavy smell of "copy-pasta"). IMHO a single window is enough that can show as well as edit the current observing list; the list selector selects the currently active list; the dialog allows all operations.

![image](https://user-images.githubusercontent.com/3529789/128021041-c1d5d20f-e4de-4452-a861-ff3c63783bb3.png)

(`(!)` = guarded, `(?)` = do we really need a close button ? maybe, if there is a save/undo feature available, otherwise ESC or the top-right close buttons are fine )

I suspect that Qt allows to type text in a dropdown list, which might be a nice way to allow the user to create a new list in an intuitive way. You see this in action in the Astrocalc dialog:
![image](https://user-images.githubusercontent.com/3529789/128022323-a574909c-c652-479d-9022-8c1766b35430.png)

(But I am prepared to admit that there might be a smell of short-sightedness to these suggestions, so I'm also looking forward to inputs of others.)